Newsapp/tvermaashutosh/README.md

```diff
@@ -0,0 +1,41 @@
+# NewsWallah
+
+NewsWallah is a news reader app built with [React](https://reactjs.org/) that uses the [New York Times API](https://developer.nytimes.com/) to fetch and display news articles.
+
+## Features
+- Browse top news stories from various categories 
```

Comment:
Attach the screenshot in it.

Reply:
@NitkarshChourasia 
I have added the screenshots in the README.md file.
Please review it again and merge it.